backup/crowbar-backup

```diff
@@ -298,10 +298,6 @@ function restore()
 
   knife index rebuild -y
 
-  # Create session store database
-  rm -f /opt/dell/crowbar_framework/db/*.sqlite3
-  su -s /bin/sh - crowbar sh -c "cd /opt/dell/crowbar_framework && RAILS_ENV=production ./bin/rake db:create db:migrate"
```

Comment:
Hrm, maybe we should keep doing the `db:migrate` bit on the last line, though (in case the backup was old, and the schema changed).